src/containers/DAO/pages/DAOPage/Proposals/CreateProposal/CreateProposalNotes.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+// - Library
+import classNames from 'classnames';
+
+// - Hooks
+import { useCreateProposal } from './hooks';
+
+// - Component
+import { FutureButton, Overlay, ConnectToWallet } from 'components';
+
+// - Styles
+import styles from './CreateProposal.module.scss';
+
+type CreateProposalNotesProps = Pick<
+	ReturnType<typeof useCreateProposal>,
+	'isLoading' | 'notes' | 'showWalletConnectModal' | 'handleHideConnectWallet'
+>;
+
+export const CreateProposalNotes: React.FC<CreateProposalNotesProps> = ({
+	notes,
+	showWalletConnectModal,
+	handleHideConnectWallet,
+}) => {
+	return (
+		<div className={styles.Notes}>
+			{/* Connect Wallet Note */}
+			{notes.ConnectWallet.show && (
+				<div className={styles.Notes_Section}>
+					<div className={styles.Notes_Section_Note}>
+						Please connect a Web3 wallet to create a proposal
+					</div>
+					<div className={styles.Notes_Section_Action}>
+						<FutureButton glow onClick={notes.ConnectWallet.onClick}>
+							Connect Wallet
+						</FutureButton>
```

Comment:
Use `ConnectWalletButton` here instead.

Also just wondering - is there a reason you've handled the show/hide "Connect Wallet" modal logic in the hook and passed values as props into this component, instead of just handling the state in here? I think it's over-complicating things here.